CustomEd.Classroom.Service/Consumers/StudentEvents/StudentUpdatedEventConsumer.cs

```diff
@@ -7,6 +7,7 @@
 
 namespace CustomEd.Classroom.Service.Consumers
 {
+    ///I think you forgot to replace teacher with student
```

Comment:
What do you mean?

Reply:
![err1](https://github.com/aenon0/CustomEd/assets/95585844/d7db59b3-90a0-436d-871e-afdb08a68a4d)

Reply:
i meant student with teacher here